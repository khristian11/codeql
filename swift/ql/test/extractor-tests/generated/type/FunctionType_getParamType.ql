import codeql.swift.elements
import TestUtils

from FunctionType x, int index
where toBeTested(x)
select x, index, x.getParamType(index)
