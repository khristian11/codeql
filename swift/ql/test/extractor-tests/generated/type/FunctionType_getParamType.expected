| ((Int) -> Int, Int) -> Int | 0 | (Int) -> Int |
| ((Int) -> Int, Int) -> Int | 1 | Int |
| (C) -> () -> () | 0 | C |
| (C.Nested) -> () -> () | 0 | C.Nested |
| (C.Nested.Type) -> () -> C.Nested | 0 | C.Nested.Type |
| (C.Type) -> () -> C | 0 | C.Type |
| (Int) -> Int | 0 | Int |
| (Int) async -> () | 0 | Int |
| (Int) async throws -> () | 0 | Int |
| (Int, Int) -> Int | 0 | Int |
| (Int, Int) -> Int | 1 | Int |
| (X.Type) -> () -> X | 0 | X.Type |
